.circleci/config.yml.jinja

```diff
@@ -12,7 +12,7 @@ jobs:
                   name: Build trace-tool
                   command: cd src/app/trace-tool && cargo build --frozen
 
-    build-wallet:
+    build-frontend:
```

Comment:
it's easier to just leave this as `build-wallet` for now. Let's let CI settle down more before changing the title here (it requires updating github settings, mergify settings, etc)